src/plugins/controls/public/options_list/embeddable/options_list_embeddable.tsx

```diff
@@ -437,14 +438,13 @@ export class OptionsListEmbeddable
 
   private buildFilter = async (): Promise<ControlFilterOutput> => {
     const {
-      componentState: { validSelections },
-      explicitInput: { existsSelected, exclude },
+      explicitInput: { existsSelected, exclude, selectedOptions },
     } = this.getState();
 
     return await this.selectionsToFilters({
       existsSelected,
       exclude,
-      selectedOptions: validSelections,
+      selectedOptions,
```

Comment:
We aren't ignoring filters anymore, so this never should have been `validSelections`